docs/ldml/tr35.md

```diff
@@ -1368,6 +1368,23 @@ root
 
 _The inheritance is often not simple truncation, as will be seen later in this section._
 
+The default search chain is slighly different for multiple variants. 
+In that case, the inheritance chain covers all combinations of variants, with longest number of variants first. 
```

Comment:
Question: How do you sort the combinations?

In your example below, how do you know "en-GB-fonipa" comes before "en-GB-scouse"?

Reply:
The ticket says "use alphabetical order". The spec should say this too.

Reply:
The spec does say so. See https://github.com/unicode-org/cldr/blob/8d829bbb16598868fc536555c4cca1c626a0d550/docs/ldml/tr35.md#5-canonicalizing-syntax

"Put any variants into alphabetical order (eg, en-fonipa-scouse, not en-scouse-fonipa)"
Do you want a pointer to that section here?